### PR TITLE
Add support for container (shell)

### PR DESCRIPTION
**Pull request by erebe**

### CODE REVIEW

File: cmd/shell.go [hunk 1]
```diff
@@ -164,24 +190,43 @@ func shellRequestWithApplicationUrl(args []string) (*pkg.ShellRequest, error) {
...
+
+		container = *containerApi
 	}
 
 	_ = pterm.DefaultTable.WithData(pterm.TableData{
 		{"Organization", string(organization.Name)},
 		{"Project", string(project.Name)},
 		{"Environment", string(environment.Name)},
 		{"Application", string(application.Name)},
+		{"Container", string(container.Name)},
```
Comment (erebe) [inline cmd/shell.go:221]: Remove this

File: cmd/console.go [hunk 2]
```diff
@@ -28,12 +28,12 @@ var consoleCmd = &cobra.Command{
 			utils.PrintlnError(err)
 			os.Exit(0)
 		}
-		application, _, err := utils.CurrentApplication()
+		service, err := utils.CurrentService()
 		if err != nil {
 			utils.PrintlnError(err)
 			os.Exit(0)
 		}
-		url := fmt.Sprintf("https://console.qovery.com/platform/organization/%v/projects/%v/environments/%v/applications/%v/summary", organization, project, environment, application)
```
Comment (erebe) [inline cmd/console.go:36]: fix this

File: pkg/shell.go [hunk 3]
```diff
@@ -14,6 +14,7 @@ import (
 const StdinBufferSize = 4096
 
 type ShellRequest struct {
+	ServiceId      utils.Id
```
Comment (mzottola) [inline pkg/shell.go:17]: I would have named it `ServiceID` to respect current format (or rename others with `Id`)

File: cmd/shell.go [hunk 4]
```diff
@@ -128,13 +141,26 @@ func shellRequestFromContext(currentContext utils.QoveryContext) (*pkg.ShellRequ
...
-		ApplicationID:  currentContext.ApplicationId,
-		ProjectID:      currentContext.ProjectId,
-		OrganizationID: currentContext.OrganizationId,
-		EnvironmentID:  currentContext.EnvironmentId,
-		ClusterID:      utils.Id(e.ClusterId),
-	}, nil
+	switch currentContext.ServiceType {
+	case utils.ApplicationType:
+		return &pkg.ShellRequest{
+			ApplicationID:  currentContext.ServiceId,
```
Comment (mzottola) [inline cmd/shell.go:147]: The `ServiceId` shouldn't be set also now ? (I suppose the `ApplicationID`  field will be removed in the future ?)

**Review by mzottola — APPROVED** [review]

File: cmd/shell.go (hunk 4 above)
Comment (erebe) [reply cmd/shell.go:147]: Indeed, it will be removed later, but for now we can't remove it

File: pkg/shell.go (hunk 3 above)
Comment (erebe) [reply pkg/shell.go]: done

**Pull request merged**